### ISSUE TITLE
transporter to planet = clouds

### ISSUE THREAD

**Issue by iceyrazor** [LemADEC/WarpDrive#507]
**Describe the bug**
--singleplayer bug
upon transporting from space to a planet. then going back to the transporter not with transporter. you can no longer use that transporter whatsoever as you get clouds while on it. when you do energize you get "transporting 0 entities" while standing on pad. unlocking, restarting, replacing block does not work. restarting your world seems to fix it.
--in multiplayer.
upon testing the same thing in multiplayer. simply relogging fixes the issue. no server restart needed. nor relocking the transporter.

**To Reproduce**
1. make a full transporter room setup in space. 
2. teleport down to planet. 
3. goback to transporter room via /space. 
4. notice clouds when stepping on transport pad and the inability to transport

**Expected behavior**
to be able to reuse the transporter without restarting my game/relogging

**Logs**
Link to pastebin/gist or equivalent logs
[latest.log](https://github.com/LemADEC/WarpDrive/files/7404106/latest.log)

**Versions** (latest is not a valid answer)
-   WarpDrive version: WarpDrive-1.12.2-1.5.23
-   Forge version: 1.12.2-14.23.5.2854
-   Modpack name/version: my own personal mod pack
-   Other notable mods & version: 

<details>
<summary>modlist</summary>
<div>same mods are used in both the singleplayer and multiplayer testing</div>
<div>AcademyCraft-1.1.3.jar</div>
<div>alchemistry-1.12.2-42.jar</div>
<div>alib-1.0.12.jar</div>
<div>astikorcarts-1.12.2-0.1.2.7.jar</div>
<div>astralsorcery-1.12.2-1.10.27.jar</div>
<div>AutoRegLib-1.3-32.jar</div>
<div>Baubles-1.12-1.5.2.jar</div>
<div>bewitchment-1.12.2-0.0.22.64.jar</div>
<div>BiblioCraft[v2.4.5][MC1.12.2].jar</div>
<div>BloodMagic-1.12.2-2.4.3-105.jar</div>
<div>Bookshelf-1.12.2-</div>
<div>2.3.590.jar</div>
<div>cc-tweaked-1.12.2-1.89.2.jar</div>
<div>Chisel-MC1.12.2-1.0.2.45.jar</div>
<div>CodeChickenLib-1.12.2-3.2.3.358-universal.jar</div>
<div>CoFHCore-1.12.2-4.6.6.1-universal.jar</div>
<div>CoFHWorld-1.12.2-1.4.0.1-universal.jar</div>
<div>CosmeticArmorReworked-1.12.2-v5a.jar</div>
<div>Craftable+Saddles[1.12]-1.3.jar</div>
<div>CreativeCore_v1.10.57_mc1.12.2.jar</div>
<div>CTM-MC1.12.2-1.0.2.31.jar</div>
<div>DoggyTalents-1.12.2-1.15.1.6.jar</div>
<div>EnchantingPlus-1.12.2-5.0.176.jar</div>
<div>EnderCore-1.12.2-0.5.73.jar</div>
<div>endercrop-1.12.2-1.6.0.jar</div>
<div>EnderIO-1.12.2-5.1.52.jar</div>
<div>Forgelin-1.8.4.jar</div>
<div>Guide-API-1.12-2.1.8-63.jar</div>
<div>Guidebook-1.12.2-2.9.5.jar</div>
<div>Hwyla-1.8.26-B41_1.12.2.jar</div>
<div>iceandfire-1.9.1-1.12.2.jar</div>
<div>InventoryTweaks-1.63.jar</div>
<div>ironchest-1.12.2-7.0.72.847.jar</div>
<div>ironfurnaces-1.3.5.jar</div>
<div>jei_1.12.2-4.15.0.296.jar</div>
<div>jetif-1.12.2-1.5.1.jar</div>
<div>journeymap-1.12.2-5.7.1.jar</div>
<div>librarianlib-1.12.2-4.22.jar</div>
<div>llibrary-1.7.20-1.12.2.jar</div>
<div>Mantle-1.12-1.3.3.55.jar</div>
<div>MineMenu-1.12.2-1.6.11-universal.jar</div>
<div>moreoverlays-1.15.1-mc1.12.2.jar</div>
<div>movingworld-1.12-6.353-full.jar</div>
<div>mysticallib-1.12.2-1.10.0.jar</div>
<div>OptiFine_1.12.2_HD_U_G5.jar</div>
<div>Patchouli-1.0-23.6.jar</div>
<div>Psi-r1.1-78.2.jar</div>
<div>RedstoneFlux-1.12-2.1.1.1-universal.jar</div>
<div>refinedstorage-1.6.16.jar</div>
<div>rpsideas-2.4.4.jar</div>
<div>SGCraft-2.0.5.jar</div>
<div>Tails-1.12-1.10.0.jar</div>
<div>TConstruct-1.12.2-2.13.0.171.jar</div>
<div>themidnight-0.3.5.jar</div>
<div>ThermalDynamics-1.12.2-2.5.6.1-universal.jar</div>
<div>ThermalExpansion-1.12.2-5.5.7.1-universal.jar</div>
<div>ThermalFoundation-1.12.2-2.6.7.1-universal.jar</div>
<div>Vampirism-1.12.2-1.6.2.jar</div>
<div>WailaHarvestability-mc1.12-1.1.12.jar</div>
<div>WarpDrive-1.12.2-1.5.23.jar</div>
<div>Wawla-1.12.2-2.5.273.jar</div>
<div>zerocore-1.12.2-0.1.2.8.jar</div>
<div>[1.12.2]DamageIndicatorsMod-3.5.1.jar</div>
</details>  

**extra info**
given the nature of this bug. i dont think its a mod conflict.

**in conclusion**
as stated above, after testing in a multiplayer, brand new world. relogging fixes this. so while this isnt that bad of a bug. it's still annoying and i felt i should report it. so low priority.

**Comment by LemADEC** [comment]
Are the angry particles on the transport pad or the core itself?
In the latest, the Transporter interface should tell you what's the issue?

**Comment by iceyrazor** [comment]
> Are the angry could particles on the transport pad or the core itself? In the latest, the Transporter interface should tell you what's the issue?

hang on, ill attach a vid

**Comment by iceyrazor** [comment]
vid
https://www.dropbox.com/s/man6se7i3p5hkfw/2021-10-24%2019-22-34.mp4?dl=0

**Comment by LemADEC** [comment]
Thanks for the video.
The "jammed" particles are on your character itself, this means there's 2+ entities in the scanner area. When it happens, type the command `/wentity 2` to see what is the second entity jamming your scanner. If you see a non-player entity, it's likely a mod adding some ghost entity, otherwise it's probably a memory leak from a mod not releasing resources properly.

**Comment by LemADEC** [comment]
I can't reproduce the issue so far on the official modpack, so it's probably related to one of your mods.
Unloading the chunk can clear such ghost entities which might explain why it comes and goes on servers, especially in dimensions that are unloading.

On a side note, whenever a key is pressed, the LUA script will refresh the display. It doesn't change how fast the core updates, it's just a display thing.

Watching the video, I've noticed a couple things regarding the scanner animation cylinder around your avatar:
1- it is centered on your feet while it should cover your whole height. That's a recurrent bug I need to fix on my side.
2- there's no visible animation on the ghost entity when you're away from the scanner, which means that entity is either very small or aligned with you constantly (which goes back to the memory leak hypothesis).

**Comment by iceyrazor** [comment]
allright ill check that out.

**Comment by iceyrazor** [comment]
> type the command `/wentity 2` to see what is the second entity jamming your scanner.

"no matching entity found within 2 blocks"

**Comment by LemADEC** [comment]
That command only detects non-players, so there's a ghost players on you, it's probably a leak from another mod.
Try to reproduce with as few mods as possible, that'll help finding where the leaks come from.

**Comment by LemADEC** [comment]
Still unable to reproduce, no update for 3 months, probably a leak in another mod, hence closing
